repos.d/apkfission.yaml

```diff
@@ -0,0 +1,40 @@
+###########################################################################
+# APK Fission packages
+#
+# https://apkfission.net/install.html
+###########################################################################
+
+{% macro apkfission(version, subrepos=['main','nonfree'], archs=['aarch64','armv7','pmmx','ppc','ppc64','x86_64']) %}
+- name: apkfission_adelie_{{version|replace('.', '_')}}
+  type: repository
+  desc: APK Fission for Adélie {{version|title}}
+  statsgroup: APKFission
+  family: apkfission
```

Comment:
I don't think this needs a dedicated family (since it's realted to adelie), also it needs a `ruleset` (to apply the same rules as for adelie). May go with the same values as adelie.

Reply:
The plan is to eventually start packaging for Alpine too; I just wanted to get all of my ducks in a row with Adélie support first.

Reply:
I agree that the Adélie repo should have the `adelie` `ruleset`; the Alpine repo can have the `alpine` one.